src/internal/common/apt.go

```diff
@@ -42,6 +42,13 @@ func getNonVirtualPackage(executor exec.Executor, name string) (pkg *AptPackage,
 	if isErrLine(execution.CombinedOut) {
 		return pkg, execution.Error()
 	}
+	
+	// Check if the output is just "Reverse Provides:" with no actual package info
+	trimmedOutput := strings.TrimSpace(execution.CombinedOut)
+	if trimmedOutput == "Reverse Provides:" {
+		return pkg, fmt.Errorf("virtual package '%s' has no concrete package providers", name)
```

Comment:
Would we lose efficiency if the code automatically gets a concrete package related to it and installs that?